config/initializers/paperclip.rb

```diff
@@ -23,5 +23,7 @@ def url(_style_name = default_style, _options = {})
     def expiring_url(_time = 3600, style_name = default_style)
       @url_generator.for(style_name, default_options)
     end
+
+    default_options[:path].gsub!(/:url/, default_options.fetch(:url))
```

Comment:
This should fix the issue described in https://github.com/on-site/grantzilla/issues/302#issuecomment-270396153